poker-spirit/hud.py

```diff
@@ -12,14 +12,33 @@
 from dotenv import load_dotenv
 
 
+class PlayerStatsWindow:
+    def merge_current_and_career_stats(self, game, career_stats):
+        '''
+        If we have the career stats and the current game stats for a given player,
+        we want to display the career stats INCLUDING the current game, so return a new
+        dict with the summed values.
+        '''
+        new_stats = {}
+        for player in game.current_players:
+            stats = game.game_stats[player]
+            new_stats[player] = defaultdict(int)
+            for stat, game_val in stats.items():
+                new_stats[player][stat] = game_val + career_stats[player][stat]
+        return new_stats
 
-class PlayerWindowsManager:
+    def populate(self, game, career_stats_by_player):
+        pass
+
+    def destroy():
+        PokerStarsGameIO
+
+class PlayerWindowsManager(PlayerStatsWindow):
```

Comment:
i feel like the manager shouldn't inherit. to me the manager is a contain of multiple windows.